docs/gaia-tutorials/join-mainnet.md

```diff
@@ -208,8 +245,4 @@ invariant broken:
 ```
 
 When submitting a invariant-broken transaction, transaction fee tokens are not
-deducted as the blockchain will halt (aka. this is a free transaction). 
-
-## Upgrade to Validator Node
```

Comment:
The Full Node setup should be providing the setup how to create a Full Node.
Opening REST API, GRPC and remote access are all suggestions that are _not_ for a validator node, which should have different security setups. Therefore upgrading an open Full Node to a Validator Node should not be emphasized in this setup.